modules/api/src/main/java/io/fluo/api/data/RowColumnValue.java

```diff
@@ -37,6 +37,10 @@ public RowColumnValue(String row, Column col, String val) {
     this.val = Bytes.of(val);
   }
 
+  public RowColumn getRowColumn() {
```

Comment:
Why add this method? Should be able to assign a RowColumnValue to a RowColumn.  Do you want to drop the value and let it be GCed?  If so, could add javadoc explaining that case.